CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+- Stop mousedown and touchstart from interrupting easeTo animations if interaction handlers are disabled ([#8725](https://github.com/mapbox/mapbox-gl-js/issues/8725))
+- Stop mouse wheel from interrupting animations when `map.scrollZoom` is disabled ([#9230](https://github.com/mapbox/mapbox-gl-js/issues/9230))
+- The previous gestures implementation prevented a camera change from actually happening if the interaction handler responsible for the change was disabled within the camera change event (`dragstart` or `rotatestart`). This behavior was not documented. It is no longer possible to prevent a camera change after it's event has been fired. It is possible some applications may be depending on this behavior. The correct way for these applications to selectively prevent camera changes is to listen to the `mousedown` or `touchstart` map event and call [.preventDefault()](https://docs.mapbox.com/mapbox-gl-js/api/#mapmouseevent#preventdefault) on it. For example, `map.on("mousedown", function(mapMouseEvent) { mapMouseEvent.preventDefault(); })`;
+- The camera change events fired by the doubleClickZoom handler (`zoomstart`, `zoom`, `zoomend`, `movestart`, `move`, `moveend`) used to have undocumented properties such as `lngLat` and `point`. These have been removed
+
+### 🍏 Other improvements
+
+
+* Line labels now have improved collision detection, with greater precision in placement, reduced memory footprint, better placement under pitched camera orientations ([#9219](https://github.com/mapbox/mapbox-gl-js/pull/9219))
+* Fix `GlyphManager` continually re-requesting missing glyph ranges ([#8027](https://github.com/mapbox/mapbox-gl-js/issues/8027), fixed by [#9375](https://github.com/mapbox/mapbox-gl-js/pull/9375)) (h/t [oterral](https://github.com/oterral))
+
+### 🐞 Bug fixes
+
+* Avoid throwing errors when calling certain popup methods before the popup element is created ([#9433](https://github.com/mapbox/mapbox-gl-js/pull/9433))
+* Fix a bug where fill-extrusion features with colinear points were not returned by `map.queryRenderedFeatures(...)` ([#9454](https://github.com/mapbox/mapbox-gl-js/pull/9454))
+* Fix a bug where using feature state on a large input could cause a stack overflow error ([#9463](https://github.com/mapbox/mapbox-gl-js/pull/9463))
+* Fix exception when using background-pattern with data driven expressions ([#9518](https://github.com/mapbox/mapbox-gl-js/issues/9518), fixed by [#9520](https://github.com/mapbox/mapbox-gl-js/pull/9520))
+* Fix a bug where UI popups were potentially leaking event listeners ([#9498](https://github.com/mapbox/mapbox-gl-js/pull/9498)) (h/t [mbell697](https://github.com/mbell697))
+* Upgrade minimist to ^1.2.5 to get fix for security issue [CVE-2020-7598](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7598) upstream ([#9425](https://github.com/mapbox/mapbox-gl-js/issues/9431), fixed by [#9425](https://github.com/mapbox/mapbox-gl-js/pull/9425)) (h/t [watson](https://github.com/watson))
```

Comment:
nit: Dependency upgrades are usually in an “Other changes” section. We can keep it here as the last bullet alternatively.

Reply:
I left it since it would be quite a lonely section for this time. But reordered it last.